api/Dockerfile

```diff
@@ -16,7 +16,6 @@ WORKDIR /app
 COPY Cargo.toml Cargo.lock ./
 RUN mkdir src; echo "fn main() {}" > src/main.rs
 RUN cargo fetch --target x86_64-unknown-linux-musl
-RUN cargo build --offline --target x86_64-unknown-linux-musl --release
```

Comment:
Why this change?

Reply:
we build later again and i am not sure about the caching, don't care performance for a release but since we run it all the time https://github.com/swissgeol/ngm/pull/834/commits/fe73733a95a7edf989d884dff798eb9ef19af84d might be better.